src/screens/HomePage/HomePage.tsx

```diff
@@ -47,6 +46,7 @@ export default function HomePage() {
   const indicatorsRef = useRef(null);
 
   const geolocation = useGeolocation();
+  const initialFipsForExplore = useGeolocationInExplore(5, geolocation);
```

Comment:
It might be better to simplify `useGeolocationInExplore` to make it always return user locations by the case of `geolocation` being `null` outside the function

```tsx
const initialFipsForExplore = geolocation
  ? useGeolocationInExplore(geolocation)
  : defaultExploreFipsLocations;
```

Reply:
per our chat, keeping all the logic+checks contained in `useGeolocationInExplore`